### PR TITLE
fix(docs): telephone number input control example is missing programa…

### LINKED ISSUE #15024: telephone number input control example is missing programatic labels
#### What is the expected behavior?
The purpose of each text input in the 'telephone number input control' example must be conveyed to users of screen reader software.

#### What is the current behavior?
The purpose of each text input in the 'telephone number input control' example is not conveyed to users of screen reader software.

#### What are the steps to reproduce?
1. Using either of the following screen readers
   - NVDA 2018.4.1 + Firefox 64
   - VoiceOver for IOS (12.1.3) + Safari
2. Open the [form field example](https://material.angular.io/components/form-field/examples)
3. Navigate to the inputs under the 'Form field with custom telephone number input control' example.
   - In NVDA you can press the tab key until you reach an input
   - In VoiceOver for iOS you can swipe right until you reach an input
4. Observe that the purpose of each of the 3 text inputs is not announced by the screen reader. They are announced as something like 'edit has auto complete'.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
- Angular Material Example 7.2.2 located at https://material.angular.io/components/
- NVDA 2018.4.1 + FireFox 64
- VoiceOver for IOS (12.1.3) + Safari (12.1.3)

#### Is there anything else we should know?
This issue can also be found by using [axe](https://www.deque.com/axe/).

### PR DESCRIPTION
**Pull request by Suresh918**
Fixes #15024

### CODE REVIEW

File: src/material-examples/form-field-custom-control/example-tel-input-example.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div [formGroup]="parts" class="example-tel-input-container">
-  <input class="example-tel-input-element" formControlName="area" size="3">
+  <input class="example-tel-input-element" formControlName="area" size="3" aria-label="Enter first 3 digits of phone number">
```
Comment (crisbeto) [inline src/material-examples/form-field-custom-control/example-tel-input-example.html]: The label should be "Area code".

File: src/material-examples/form-field-custom-control/example-tel-input-example.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   <span class="example-tel-input-spacer">&ndash;</span>
-  <input class="example-tel-input-element" formControlName="exchange" size="3">
+  <input class="example-tel-input-element" formControlName="exchange" size="3" aria-label="Enter middle 3 digits of phone number">
```
Comment (crisbeto) [inline src/material-examples/form-field-custom-control/example-tel-input-example.html]: The label should be "Exchange code".

File: src/material-examples/form-field-custom-control/example-tel-input-example.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
   <span class="example-tel-input-spacer">&ndash;</span>
-  <input class="example-tel-input-element" formControlName="subscriber" size="4">
+  <input class="example-tel-input-element" formControlName="subscriber" size="4" aria-label="Enter last 4 digits of phone number">
```
Comment (crisbeto) [inline src/material-examples/form-field-custom-control/example-tel-input-example.html]: The label should be "Subscriber number".

File: src/material-examples/form-field-custom-control/example-tel-input-example.html (hunk 1 above)
Comment (Suresh918) [reply src/material-examples/form-field-custom-control/example-tel-input-example.html]: Changed as per your suggestion. Can you please check now.

File: src/material-examples/form-field-custom-control/example-tel-input-example.html (hunk 2 above)
Comment (Suresh918) [reply src/material-examples/form-field-custom-control/example-tel-input-example.html]: Changed as per your suggestion. Can you please check now

File: src/material-examples/form-field-custom-control/example-tel-input-example.html (hunk 3 above)
Comment (Suresh918) [reply src/material-examples/form-field-custom-control/example-tel-input-example.html]: Changed as per your suggestion. Can you please check now

**Comment by jelbourn** [comment]
@Suresh918 can you rebase this PR?

**Comment by Suresh918** [comment]
@jelbourn Rebased the pull request

**Pull request merged**